README.rst

```diff
@@ -37,7 +37,8 @@ REST API:
 
     $stack = HandlerStack::create();
 
-    $middleware = new Oauth1([
+    $middleware = new Oauth1();
```

Comment:
This is wrong: to pass an array to constructor is still required.

This is the correct way:

```php
$middleware = new Oauth1([]);
```